host/bcm283x/gpio.go

```diff
@@ -479,11 +480,57 @@ func (p *Pin) PWM(duty gpio.Duty, period time.Duration) error {
 		old := pwmMemory.ctl
 		pwmMemory.ctl = (old & ^(0xff << shift)) | ((pwm1Enable | pwm1MS) << shift)
 	}
-
+	p.usingClock = true
 	p.setFunction(f)
 	return nil
 }
 
+// StreamIn implements gpiostream.PinIn.
+//
+// DMA driven StreamOut is avaiable for GPIO0 to GPIO31 pin and the resolution is
+// 200KHz.
+func (p *Pin) StreamIn(pull gpio.Pull, s gpiostream.Stream) error {
+	b, ok := s.(*gpiostream.BitStreamLSB)
+	if !ok {
+		return errors.New("bcm283x: other Stream than BitStreamLSB are not implemented")
+	}
+	if err := p.In(pull, gpio.NoEdge); err != nil {
+		return err
+	}
+	if err := dmaReadStream(p, b); err != nil {
+		return p.wrap(err)
+	}
+	return nil
+}
+
+// StreamOut implements gpiostream.PinOut.
+//
+// PCM/I2S driven StreamOut is available for GPIO21 pin. The resolution is up to
+// 250MHz.
+//
+// For GPIO0 to GPIO31 except GPIO21 pin, DMA driven StreamOut is avaiable and
```

Comment:
available*
same comment about maximum and kHz.

Reply:
Done.